src/list-data/conversion-types.ts

```diff
@@ -43,5 +43,25 @@ export const ConversionTypes: ConversionTypesI[] = [
     ],
     value: NrRequestTypeCodes.CONVERT_CORP,
     shortlist: false
+  },
+  {
+    desc: 'Unlimited Liability Company to a limited Company',
+    text: 'Unlimited Liability Company to a limited Company',
+    entity_type_cd: EntityType.CR,
+    blurbs: [
+      'Alter business type from an unlimited Liability Company to a limited Company.'
+    ],
+    value: NrRequestTypeCodes.CONVERT_ULCB,
+    shortlist: false
+  },
+  {
+    desc: 'Unlimited Liability Company to a Benefit Company',
+    text: 'Unlimited Liability Company to a Benefit Company',
+    entity_type_cd: EntityType.BC,
+    blurbs: [
+      'Alter business type from an ULC to a Benefit Company.'
+    ],
+    value: NrRequestTypeCodes.CONVERT_ULBE,
+    shortlist: false
```

Comment:
By the way, does anyone know why these 2 records were deleted? Mihai/Janis think they should be there, which is why I restored them.

Reply:
I was wondering why were they deleted in the first place. No idea Sev.

Reply:
Ah, this is why: https://github.com/bcgov/namerequest/pull/621

Reply:
Oh, this is the commit that the branch is behind on in regards to main. OK, makes sense.